<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPServer</class>
 <widget class="QDialog" name="UDPServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDPServer</string>
  </property>
  <widget class="QPushButton" name="LED">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>124</width>
     <height>305</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#LED {
background-color: transparent;
border-image: url(:images/LEDOFF.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="EXIT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#EXIT {
background-color: transparent;
border-image: url(:images/EXIT.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="HELLO">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#HELLO {
background-color: transparent;
border-image: url(:images/say-hello.png);
background: none;
border: none;
background-repeat: none;
}
#HELLO:pressed
{
border-image: url(:images/say-hello.png);
}
#HELLO:onClicked
{
border-image: url(:images/say-hello.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="TOGGLE">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox#TOGGLE {font-size:14pt;font-weight:bold;color: rgb(93,94,95)}
#QCheckBox#TOGGLE::indicator{ background-color:#FFF; }
#QCheckBox#TOGGLE{border: 2px solid #5A5A5A;background: none;}

</string>
   </property>
   <property name="text">
    <string>Toggle</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>108</width>
     <height>214</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="client">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Arial&quot;;
color: rgb(255, 0, 0);

text-decoration: underline;

</string>
        </property>
        <property name="text">
         <string>Client:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="message_to_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#message_to {font-size:11pt;font-weight:bold;color:#550000;}</string>
        </property>
        <property name="text">
         <string>Message to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="port_to_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QLabel#port_to {font-size:11pt;font-weight:bold;color:#550000;}
</string>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="data_to_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#data_to {font-size:11pt;font-weight:bold;color:#550000;}</string>
        </property>
        <property name="text">
         <string>Data:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="Server">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;
color: rgb(255, 0, 0);

text-decoration: underline;
</string>
      </property>
      <property name="text">
       <string>Server:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="message_from_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#message_from {font-size:11pt;font-weight:bold;color:#550000;}</string>
        </property>
        <property name="text">
         <string>Message from:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="port_from_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#port_from {font-size:11pt;font-weight:bold;color:#550000;}</string>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="data_from_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#data_from {font-size:11pt;font-weight:bold;color:#550000;}</string>
        </property>
        <property name="text">
         <string>Data:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="display_message_to">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#display_message_to {
background-color: white;
opacity: 0.5;
margin: 10px;
font-weight:bold;
border: none;
background-repeat: none;
font-color:blue;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="display_port_to">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#display_port_to {
background: white;
font-weight:bold;
border: none;
background-repeat: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="display_data_to">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#display_data_to {
background: white;
font-weight:bold;
border: none;
background-repeat: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="display_message_from">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="display_port_from">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="display_data_from">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
